### PR TITLE
ON-1848 Changes to Manage Users page

### PR DESCRIPTION
**Pull request by davidgouge**
Change user name to link to a (currently non-existant) User details page.
Added button to (currently non-existant) Invite User page

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Web/Controllers/ManageUsers/InviteUserController.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+using SFA.DAS.AssessorService.Web.Infrastructure;
+
+namespace SFA.DAS.AssessorService.Web.Controllers.ManageUsers
+{
+    [Authorize]
+    [CheckSession]
+    public class InviteUserController : Controller
+    {
+        public InviteUserController()
+        {
```
Comment (martin-hornsby) [inline src/SFA.DAS.AssessorService.Web/Controllers/ManageUsers/InviteUserController.cs]: I assume there will be some sort of implementation in here later 😉

File: src/SFA.DAS.AssessorService.Web/Views/ManageUsers/Index.cshtml [hunk 2]
```diff
@@ -31,29 +31,30 @@
                         </tr>
```
Comment (martin-hornsby) [inline src/SFA.DAS.AssessorService.Web/Views/ManageUsers/Index.cshtml]: Whilst not part of this PR, we're exposing and API Type directly to the view. This should be a View Model

File: src/SFA.DAS.AssessorService.Web/Controllers/ManageUsers/InviteUserController.cs (hunk 1 above)
Comment (davidgouge) [reply src/SFA.DAS.AssessorService.Web/Controllers/ManageUsers/InviteUserController.cs]: Yes, as I mentioned, the implementation is in a different ticket.  This is just for the main page.

File: src/SFA.DAS.AssessorService.Web/Views/ManageUsers/Index.cshtml (hunk 2 above)
Comment (davidgouge) [reply src/SFA.DAS.AssessorService.Web/Views/ManageUsers/Index.cshtml]: On the whole, I agree, but in this case the VM and the type would be identical and it would be a map between the two for no reason.  If that part of the view becomes more complicated and view specific, then yeah, VM all the way.

File: src/SFA.DAS.AssessorService.Web/Views/ManageUsers/Index.cshtml [hunk 3]
```diff
@@ -31,29 +31,30 @@
...
                     <tbody>
-                        @foreach (var item in Model)
+                        @foreach (var contact in Model)
                         {
                             <tr>
-                                <td data-label="@Localizer["User details"]" class="nowrap" >
-                                    @Html.DisplayFor(modelItem => item.Contact.DisplayName)<br />
-                                    <span>@Html.DisplayFor(modelItem => item.Contact.Email)</span>
+                                <td data-label="@Localizer["User details"]" class="nowrap">
+                                    <a asp-controller="ManageUsers" asp-action="User" asp-route-userid="@contact.Contact.Id">@Html.DisplayFor(modelItem => contact.Contact.DisplayName)</a>
```
Comment (gdbarnes) [inline src/SFA.DAS.AssessorService.Web/Views/ManageUsers/Index.cshtml]: This will show the new link correctly when the css upgrade is merged:
```suggestion
                                    <a class="govuk-link" asp-controller="ManageUsers" asp-action="User" asp-route-userid="@contact.Contact.Id">@Html.DisplayFor(modelItem => contact.Contact.DisplayName)</a>
```

File: src/SFA.DAS.AssessorService.Web/Views/ManageUsers/Index.cshtml [hunk 4]
```diff
@@ -69,6 +70,7 @@
                     </div>
                 </div>
             }
+            <a asp-controller="InviteUser" asp-action="Invite" class="button">Invite new user</a>
```
Comment (gdbarnes) [inline src/SFA.DAS.AssessorService.Web/Views/ManageUsers/Index.cshtml]: This will show the button correctly when the css upgrade is merged:
```suggestion
            <a asp-controller="InviteUser" asp-action="Invite" class="button govuk-button">Invite new user</a>
```
